pkgs/development/python-modules/snakemake/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  stdenv
+, buildPythonPackage
+, fetchPypi
+, appdirs
+, ConfigArgParse
+, datrie
+, docutils
+, jsonschema
+, pyyaml
+, ratelimiter
+, requests
+, wrapt
+}:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "snakemake";
+  version = "5.2.2";
+
+  propagatedBuildInputs = [
+    appdirs
+    ConfigArgParse
+    datrie
+    docutils
+    jsonschema
+    pyyaml
+    ratelimiter
+    requests
+    wrapt
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "adffe7e24b4a613a9e8bf0a2a320b3cea236d86afb9132bb0bbbc08b8e35a3a3";
+  };
+
+  doCheck = false; # Tests depend on Google Cloud credentials at ${HOME}/gcloud-service-key.json
+
+  meta = with stdenv.lib; {
+    homepage = "http://snakemake.bitbucket.io";
+    license = "MIT";
+    description = "A workflow management system that aims to reduce the complexity of creating workflows by providing a fast and comfortable execution environment, together with a clean and modern specification language in python style. Snakemake workflows are essentially Python scripts extended by declarative code to define rules. Rules describe how to create output files from input files";
```

Comment:
This is too long for description, move it to `longDescription` and use something shorter for `description`. You can get inspiration on [Repology](https://repology.org/metapackage/snakemake/packages).